sentry-samples/sentry-samples-spring/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+# NOTE: Replace the test DSN below with YOUR OWN DSN to see the events from this app in your Sentry project/dashboard
+sentry.dsn=https://f7f320d5c3a54709be7b28e0f2ca7081@sentry.io/1808954
+sentry.send-default-pii=true
+
+# Uncaught exception handler should be disabled if both Sentry Spring and Logback integrations are enabled
```

Comment:
just out of curiosity, why? do they already register an uncaught exception handler? should we not do this automatically then? otherwise, customers would need to set extra properties manually.

Reply:
If there is no uncaught exception handler, exception is just logged with the logging framework. If its turn on in both then we have duplicated event.

Reply:
does that mean we should then disable it automatically somehow? I see this being a constant support ticket -> why dow e get duplicated events?

the other thing would be, we rely on our uncaught exception handler to end a `session` automatically, if we disable it, we'd need to find another way, or the current session would never be ended and marked as crashed properly.
I'd suggest creating a new issue here https://github.com/getsentry/sentry-java/projects/1 so we decide where/how to proceed

Reply:
Flag is gone. This is solved now by event deduplication.